.circleci/config.yml

```diff
@@ -25,6 +25,9 @@ jobs:
             SCHEME: JWTDecode-iOS
             DEVICE: iPhone 8
             FASTLANE_EXPLICIT_OPEN_SIMULATOR: 2
+      - run:
+          name: SwiftPM Test
+          command: swift test
```

Comment:
Let's rename this to `Swift Package Manager Test`. No need to save chars here, the more explicit the better. 

Also, I'm trying to understand if we can lower the build run time instead of increasing it. Is this a compliment command or is it supposed to replace another command already present on this script?

Reply:
This is a complementary command meant to ensure that the package can be built and tested with SPM.